hedera-node/src/test/java/com/hedera/services/store/schedule/HederaScheduleStoreTest.java

```diff
@@ -148,6 +151,38 @@ public void setup() {
         subject.setHederaLedger(hederaLedger);
     }
 
+    @Test
+    void rebuildsAsExpected() {
+        // setup:
+		String reconnectMemo = "Back again!";
+        ArgumentCaptor<BiConsumer<MerkleEntityId, MerkleSchedule>> captor = forClass(BiConsumer.class);
+        MerkleSchedule reconnectSchedule = new MerkleSchedule(
+                transactionBody,
+                EntityId.ofNullableAccountId(IdUtils.asAccount("1.2.3")),
+                RichInstant.MISSING_INSTANT);
+        reconnectSchedule.setMemo(reconnectMemo);
```

Comment:
We set the memo here, we should validate it later too, right?

Reply:
Since the `memo` is a value component of the `ContentAddressableSchedule` created on line 165, the `containsKey` assert on line 182 validates that all relevant fields of `reconnectSchedule` were used in `buildContentAddressableViewOfExtantSchedules()`.